### ISSUE TITLE
uncaught exception: Error: Permission denied for <http://rpl.host> to get property Proxy.InstallTrigger

### ISSUE THREAD

**Issue by maxgu** [dimikot/dklab_realplexor#11]
это при инициализации realplexor.execute();
как ловить такой эксепшен?
и можно ли вообще?

З.Ы. FF 8.0

**Comment by dimikot** [comment]
Что такое rpl.host?

**Comment by maxgu** [comment]
ну это я так здесь хост назвал, там  имя хоста, гле поднят реаплексор.
он работает, но не сразу, первый запрос не работает из-за это эксепшена, подозреваю, а потом норм

**Comment by dimikot** [comment]
Вообще, Proxy.InstallTrigger - в интернетах пишут, что это баги Firebug. Я тоже несколько раз натыкался на это сообщение (кстати, в случаях, не связанных никак с реалплексором). Думаю, что и в данном случае такая же ситуация.

Уверены, что первый запрос именно отваливается? На вкладке "Сеть" в firebug-е это видно? А в других браузерах?

**Comment by maxgu** [comment]
да, это бага ФФ, вроде бы возникает, когда скриптом создается iframe
просто я думал может вы знаете как словить такой эксепшен.

да, после возникновения эксепшена JS код дальше не выполняется ... пока решил с помощью settimeout() через 300 ms подключаюсь снова и все работает, т.к. первый заканчивается эксепшеном.
